content/TrueCommand/SystemManagement/_index.md

```diff
@@ -4,7 +4,7 @@ weight: 30
 geekdocCollapseSection: true
 ---
 
-The TrueCommand dashboard gives quick status overviews of the TrueNAS systems it is connected to.
+The TrueCommand dashboard provides status overviews of each connected TrueNAS system.
 
 ![TrueCommandDashboard](/images/TrueCommand/2.0/TCDashBoard.png "TrueCommand Dashboard")
```

Comment:
Remove the top menu bar from the screenshot to use with 2.1 changes this image does not include. Otherwise the image is fine.